samples/bot-sso-adaptivecard/nodejs/resources/AdaptiveCardWithSSOInRefresh.json

```diff
@@ -0,0 +1,36 @@
+﻿{
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "version": "1.5",
```

Comment:
when are you responding with this card with sso options in the bot flow?

Reply:
comment fixed